### ISSUE TITLE
Creating Quick Fix Challenges following the instructions of the wiki does not work

### ISSUE THREAD

**Issue by ENT8R** [osmlab/maproulette3#945]
I wanted to test the new quick fix feature which has been introduced recently. I followed the instructions at https://github.com/osmlab/maproulette3/wiki/Creating-Quick-Fix-Challenges and added the data to the GeoJSON file that I wanted to upload.
The content of the file follows the exact same schema of the wiki instructions, e.g.:
<details>
  <summary>Click to view example</summary>
  
This is an example of a feature in the FeatureCollection of the GeoJSON file:
```json
{
  "type": "Feature",
  "properties": {
	"@id": "way/27618526",
	"addr:city": "Burgdorf",
	"addr:housenumber": "27",
	"addr:postcode": "31303",
	"addr:street": "Marktstraße",
	"building": "yes",
	"name": "REWE",
	"opening_hours": "Mo-Sa 07:00-21:30",
	"phone": "05136-9723516",
	"shop": "supermarket",
	"website": "https://www.rewe.de",
	"wheelchair": "yes",
	"maproulette": {
	  "suggestedFix": {
		"meta": {
		  "version": 1
		},
		"operations": [
		  {
			"operationType": "modifyElement",
			"data": {
			  "id": "way/27618526",
			  "operations": [
				{
				  "operation": "setTags",
				  "data": {
					"phone": "+49 5136 9723516"
				  }
				}
			  ]
			}
		  }
		]
	  }
	}
  },
  "geometry": {
	"type": "Polygon",
	"coordinates": [
	  [
		[
		  10.0126533,
		  52.4470812
		],
		[
		  10.0121364,
		  52.4469855
		],
		[
		  10.0122067,
		  52.4468509
		],
		[
		  10.012156,
		  52.4468429
		],
		[
		  10.0124063,
		  52.4463693
		],
		[
		  10.0127934,
		  52.4464518
		],
		[
		  10.0128664,
		  52.4464649
		],
		[
		  10.0126746,
		  52.4468549
		],
		[
		  10.0127646,
		  52.4468688
		],
		[
		  10.0126584,
		  52.447067
		],
		[
		  10.0126357,
		  52.4470658
		],
		[
		  10.0126533,
		  52.4470812
		]
	  ]
	]
  }
}
```
</details>

I then uploaded the GeoJSON file to MapRoulette via the web interface, started the challenge, but there was no button for applying the suggested fix, just the other buttons allowing me to normally edit the feature in an editor of my choice...

Are the instructions of the wiki 100% right or am I missing something? Is it right that the `maproulette` property has to be under `feature.properties`?

**Comment by ENT8R** [comment]
I tried now to convert the GeoJSON file into the line by line format (https://github.com/osmlab/maproulette3/wiki/Line-by-Line-GeoJSON-Format) and now it seems to be working... I changed nothing else, so is it intentionally that it should not work when not using the line by line format?

**Comment by nrotstan** [comment]
Thanks for the report, @ENT8R , it’s possible there is a bug. Would you mind posting your geojson that didn’t work so we can take a look?

**Comment by ENT8R** [comment]
Thank you for your quick response! I uploaded the whole file to https://gist.github.com/ENT8R/72125f7427af03576da3699c347f35ed
This one is a pretty big file with about 4500 different features, but the same bug is also reproducible with the first 25 entries, so here is another one: https://gist.github.com/ENT8R/269493139360e4cb9124c89e56a5c5d0

**Comment by nrotstan** [comment]
I'm able to replicate the issue with that geojson and can confirm that this is due to a MR bug.

**Comment by nrotstan** [comment]
Included in backend release 3.6.7 (but not yet deployed to maproulette.org)

**Comment by nrotstan** [comment]
This should now be fixed on maproulette.org
